decentralised-identity.md

```diff
@@ -0,0 +1,157 @@
+
+- What data needs to be collected by DID and why?
+- How is this data going to be used and how could systems like this mitigate the risks?
+- Why it's a bad idea to store your internet data with your personal data?
+- Should our personal data and identity be kept off the internet / blockchain entirely and why?
+- What do protocol providers have to gain from building these platforms and collecting your data?
+- What is the potential for backdoor Corp / Gov keys?
+- What is the potential for your data to be harvested or sold on these new systems?
+- What are the alternatives to storing your data?
+- Can a data-less system be created for our internet persona?
+- Why is it a bad idea to store personal data on-chain and what are the long term implementations?
+
+Government-issued documents like passports, driver's licenses, and birth certificates are already widely accessible, raising concerns about the security and privacy of personal data. While addressing these issues at a government level is crucial, the idea of connecting personal data to online activities poses a significant risk to anonymity, privacy, and security. The vision here is to keep these entirely separate – a system where leaking internet data would leave no trace of one's real identity. Embracing cypherpunk principles means scrutinizing any system that seeks personal data, as the link between personal and internet data undermines the very values of anonymity and privacy the cypherpunk movement advocates for.
+
+### Conclusion:
+
+In summary, decentralized identity aligns with the cypherpunk philosophy by promoting user autonomy, privacy, and security. It envisions a future where individuals have more control over their digital identities. Imagine having a digital identity that's flexible, private, and works seamlessly across various online platforms. It's like having a key that fits perfectly into every digital lock you encounter.
+
+Moreover, the interoperability of decentralized identity ensures that your digital persona isn't confined to a single platform – it travels with you. By eliminating single points of failure, the system becomes more resilient and less vulnerable to large-scale breaches.
+
+In a user-centric world, you decide how, when, and where to use your digital identity. This cypherpunk vision pushes us towards a future where individuals hold the reins of their digital lives, supported by secure, private, and user-friendly decentralized identity systems.
+
+However, it's crucial to acknowledge that this utopian vision of DID is not without its challenges. The potential drawbacks include concerns about scalability, energy consumption, and the complex integration of blockchain technology. Achieving widespread adoption and navigating legal and regulatory uncertainties also present ongoing hurdles. 
+
+In conclusion, while decentralized identity holds immense promise, addressing these challenges is imperative to ensure a secure, private, and user-friendly evolution.
+
```

Comment:
I've added a few more points, nothing major, but they might need to be cleaned up for article consistence. Once these are address you are good to build on CT. Let me know when it's done and I'll schedule for publishing.

Reply:
Thanks again for feedback.